src/categorical_algebra/DataMigrations.jl

```diff
@@ -438,9 +383,24 @@ function colimit_representables(F::DeltaSchemaMigration, y)
 end
 function colimit_representables(F::ConjSchemaMigration, y)
   C = dom(F)
+  ACS = constructor(ob_map(y,first(ob_generators(dom(y)))))
   colimits = make_map(ob_generators(C)) do c
-    Fc = ob_map(F, c)
-    colimit(compose(op(Fc), y))
+    Fc = ob_map(F, c) # e.g. I
+    clim_diag = deepcopy(compose(op(Fc), y))
+    # modify the diagram we take a colimit of to concretize some vars
+    params = Fc isa SimpleDiagram ? Dict() : Fc.params
+    G = dom(clim_diag.diagram).cat.graph
```

Comment:
Use the accessor functions to get these rather than digging through the fields, which again are a private implementation detail.